adaptors/backbone/context_adaptor.js

```diff
@@ -65,7 +65,7 @@ var lookupValue = function(view, name) {
     return view.at(name);
   }
   // Fall back to a property check
-  if (view[name] != null) {
```

Comment:
this will break some stuff unfortunately.. since computed properties are still in use internally, they rely on accessing the function from the model prototype, which this will now negate.

maybe something like `if (view[name] != null && typeof view === 'object')`?

could also do something like `if (view[name] != null && isNotNativeFunction(view[name]))`
with `isNotNativeFunction` using something like [this](https://davidwalsh.name/detect-native-function)